examples/hello-world/src/main/kotlin/com/justai/jaicf/examples/helloworld/channel/Telegram.kt

```diff
@@ -4,5 +4,5 @@ import com.justai.jaicf.channel.telegram.TelegramChannel
 import com.justai.jaicf.examples.helloworld.helloWorldBot
 
 fun main() {
-    TelegramChannel(helloWorldBot, "580468601:AAHaMg4gOsN2A_zvIO6-PouVk3GcZ_WVrdI").run()
+    TelegramChannel(helloWorldBot, "580468601:AAFZaIXLiKbD7XRsGVHX74N-5nlXbttvEzc").run()
```

Comment:
Is it a real token?
Maybe we should host a bot somewhere (heroku/jaicp cloud) with this token to be a demo for users, what do you think?

Reply:
I don't think so